wimm.Missionary/ConversionMap.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace wimm.Missionary
+{
+    /// <inheritDoc />
+    public class ConversionMap<T> : IConversionMap<T>
+    {
+        private readonly IDictionary<Type, object> _conversions = new Dictionary<Type, object>();
+
+        public void Set<U>(IConversion<T, U> conversion) =>
+            _conversions[typeof(U)] = (conversion ?? throw new ArgumentNullException(nameof(conversion)));
+
+        public IConversion<T, U> Get<U>() =>
+            _conversions.TryGetValue(typeof(U), out var conversion)
+                ? (IConversion<T, U>)conversion
+                : null;
```

Comment:
How many times do you have to write a maybe type before you start using it :P

Reply:
After I get this lib to 1.0 I'll publish a Secundatives with just the Maybe in it